alphadia/outputtransform.py

```diff
@@ -820,16 +819,19 @@ def build_library(
 
 
 def _build_run_stat_df(
+    folder: str,
     raw_name: str,
     run_df: pd.DataFrame,
-    optimization_manager_path: str,
     channels: list[int] | None = None,
 ):
     """Build stat dataframe for a single run.
 
     Parameters
     ----------
 
+    folder: str
+        Directory containing the raw file and the managers
```

Comment:
is this a folder name? a relative path? an absolute path? please add this information to the docstring and conaider changing also the name (e.g.  to`output_folder_path`)